staging/src/k8s.io/apiserver/pkg/storage/etcd3/compact.go

```diff
@@ -160,3 +163,16 @@ func compact(ctx context.Context, client *clientv3.Client, t, rev int64) (int64,
 	klog.V(4).Infof("etcd: compacted rev (%d), endpoints (%v)", rev, client.Endpoints())
 	return curTime, curRev, nil
 }
+
+func CompactTestStorage(etcdClient *clientv3.Client) storagetesting.Compaction {
```

Comment:
PTAL at - it doesn't seem to be needed:
https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/apiserver/pkg/storage/etcd3/store_test.go#L254

Reply:
I meant to replace `compactStorage` with `CompactTestStorage`.

Reply:
Done